src/components/ListCryptos.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { BsSearch } from 'react-icons/bs';
+import { FaChevronDown, FaChevronUp } from 'react-icons/fa';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchCrypto } from '../redux/features/Cryptocurrency/CryptoSlice';
+
+const ListCryptos = () => {
```

Comment:
- [ ] You did an amazing job with the search functionality. :+1: Let's work on some styling issues to match the design guideline. ✍️ Kindly reduce both the horizontal and vertical spacing between the cards. I believe the color variation is enough, there is no need to have space between the cards. 

| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/katfogy/react-capstone-web-metric-app/assets/17243325/19580dd0-efc7-4bce-86cd-e84fe8bffafc" width="300">|<img src="https://github.com/katfogy/react-capstone-web-metric-app/assets/17243325/623b9581-b513-40fa-bd15-ef869472eb47" width="300">|